app/views/courses/_form.html.erb

```diff
@@ -32,27 +30,23 @@
         <tr class='row'>
           <th class="col-sm-2">CRN</th>
           <th class="col-sm-4">Instructor CCS Username</th>
-          <th class="col-sm-6">Meeting time</th>
+          <th class="col-sm-5">Meeting time</th>
+          <th class="col-sm-1">Delete</th>
```

Comment:
This is scary, if you're editing the course after students have started registering.  What will you do with those registrations?